component/common/src/main/java/org/wso2/carbon/identity/smsotp/common/SMSOTPServiceImpl.java

```diff
@@ -276,6 +279,8 @@ private void triggerNotification(User user, String otp) throws SMSOTPException {
         }
 
         HashMap<String, Object> properties = new HashMap<>();
+        String uniqueId = getCorrelationId();
+        properties.put(Constants.UNIQUE_ID, uniqueId);
```

Comment:
Should be a Map Object. 
``` Map<String, Object> properties = new HashMap<>();```

Can use Java 8 Map initializations,
```Map<String, Object> properties = new HashMap<>(){{
put(key1, value1);
put(key2, value2);
}}```

Java 8 Streams,
```Stream.of()```

Reply:
This is a suggestion, clarify it.

Reply:
In this case, we will use `Map` because connectors like this are used in all IS versions. So, if we use stream.of(), it will not support it in older versions where JAVA 7 or below is used.